src/main/java/nl/scheveschilder/techiteasybackend/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,122 @@
+package nl.scheveschilder.techiteasybackend.controllers;
+
+import nl.scheveschilder.techiteasybackend.exceptions.RecordNotFoundException;
+import nl.scheveschilder.techiteasybackend.models.Television;
+import nl.scheveschilder.techiteasybackend.repositories.TelevisionRepository;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/tv")
+public class TelevisionsController {
+
+    private final TelevisionRepository repos;
+
+    public TelevisionsController(TelevisionRepository repos) {
+        this.repos = repos;
+    }
+
+
+    @GetMapping("/all")
+    public ResponseEntity<List<Television>> getTelevisions(){
+        if (repos.findAll().isEmpty()) {
```

Comment:
Ik snap dat je de check doet, echter als de lijst leeg is mag je de lege lijst returnen naar de gebruiker. de 204 (no content) is echt bedoeld als je zeker weet dat er geen data terug kan komen. bijvoorbeeld bij het verwijderen, Het is gelukt, maar ik heb geen data, dus krijg je een 204, In dit geval kan de lijst aangepast worden. dus zou je de laatste return juist zijn. 

(ook in dit geval doe je 2x een call naar de database (je kan ook de televisies opslaan in een lijst en dan verder met die lijst kijken en returnen als dat nodig is :) )